core/src/main/java/nu/yona/server/messaging/service/DiscloseRequestMessageDTO.java

```diff
@@ -77,8 +71,9 @@ public Status getStatus()
 	public static DiscloseRequestMessageDTO createInstance(UserDTO actingUser, DiscloseRequestMessage messageEntity)
 	{
 		GoalConflictMessage targetGoalConflictMessage = messageEntity.getTargetGoalConflictMessage();
-		return new DiscloseRequestMessageDTO(messageEntity.getID(), messageEntity.getCreationTime(), messageEntity.getNickname(),
-				messageEntity.getStatus(), targetGoalConflictMessage.getOriginGoalConflictMessageID(),
+		return new DiscloseRequestMessageDTO(messageEntity.getID(), messageEntity.getCreationTime(), actingUser,
```

Comment:
I don't agree. The message still needs to be rendered after the user sending this message does no longer exist... So I guess the `UserDTO.createInstanceIfNotNull(messageEntity.getUser())` is the only correct solution.